pkg/machine/qemu/machine.go

```diff
@@ -1749,6 +1798,10 @@ func (p *Provider) VMType() string {
 	return vmtype
 }
 
+func toVlanSockPath(path string) string {
```

Comment:
Will not be needed, when https://github.com/containers/podman/issues/15852 is implemented. Now there is nowhere to save this, so, we rely on a different saved value and derive from it.